	This is file patch-07-dist for pkg-config-0.21
		to be applied on top of patch-05-automake-warning

	Don't distribute DISTCLEAN files

diff -ur -x autom4te.cache -x aclocal.m4 -x configure -x Makefile.in -x config.h.in -x 'config.h.in~' pkg-config-0.21.orig/glib-1.2.8/Makefile.am pkg-config-0.21/glib-1.2.8/Makefile.am
--- glib-1.2.8/Makefile.am	2007-01-19 15:03:19.000000000 +0100
+++ glib-1.2.8/Makefile.am	2007-01-19 15:07:29.000000000 +0100
@@ -23,9 +23,7 @@
 	makefile.msc.in		\
 	makefile.msc		\
 	giowin32.c		\
-	glibconfig.h.win32	\
 	glibconfig.h.win32.in	\
-	config.h.win32		\
 	config.h.win32.in
 
 noinst_LTLIBRARIES = libglib.la
